digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -73,7 +73,7 @@ case class RoadAddress(id: Long, roadNumber: Long, roadPartNumber: Long, track:
                        calibrationPoints: (Option[CalibrationPoint], Option[CalibrationPoint]) = (None, None), floating: Boolean = false,
                        geom: Seq[Point])
 
-case class RoadAddressProject(id: Long, status: Long, name: String, createdBy: String, createdDate: DateTime, modifiedBy:String, startDate: DateTime, dateModified: DateTime, additionalInfo :String, roadNumber: Long, startPart: Long, endPart: Long)
+case class RoadAddressProject(id: Long, status: Long, name: String, createdBy: String, createdDate: DateTime, modifiedBy:String, startDate: DateTime, dateModified: DateTime, additionalInfo :String,list: List[minRoadAddressPart]) //List[(roadNumber: Long, startPart: Long, endPart: Long])
```

Comment:
"//List[(roadNumber: Long, startPart: Long, endPart: Long]) 
"Its better to leave this commented code here, or can be removed?